src/admin/components/ResourceTabs/ChangeRequestsTab/useFormChanges.ts

```diff
@@ -51,7 +50,7 @@ export default function useFormChanges(current: any, changed: any, steps: FormSt
               title: entry.title,
               type: entry.type,
               currentValue,
-              newValue: isEqual(currentValue, newValue) ? undefined : newValue
+              newValue: currentValue === newValue ? undefined : newValue
```

Comment:
This deep equals check was intentional, and I believe this change will break existing functionality.

Reply:
Thank you for your input. I reverted the change to maintain the deep equals check as it was originally implemented in the staging branch to avoid breaking the existing functionality.